Makefile

```diff
@@ -214,7 +215,7 @@ include filesystem.mk
 
 # TODO: Rules for Pearl
 # FIXME: Computed secure area CRC in header is incorrect due to first 8 bytes of header not actually being "encryObj"
-$(ROM): rom.rsf arm9 arm7 filesystem $(BNR) tools/bin/rom_header.template.sbin
+$(ROM): rom.rsf filesystem arm9 arm7 $(BNR) tools/bin/rom_header.template.sbin
```

Comment:
filesystem should be a dependancy for arm9 and arm7 and not just in order, as this will break -j